### ISSUE TITLE
Any option to "disable“ margin or do not draw the margin rectangle if SetMargins(0, 0, 0, 0)

### ISSUE THREAD

**Issue by asmwarrior** [GitHubLionel/wxMathPlot#15]
Hi, I try to minimize the margin area by calling such function: `SetMargins(0, 0, 0, 0);`, but I see there is still a black rectangle box drawn around the border of the mpWindow. I mean is it possible to disable drawing of such rectangle box?

Thanks.

**Comment by asmwarrior** [comment]
I may be wrong. I see that by default, there are two rectangles around the margin area. When I set the `SetMargins(0, 0, 0, 0);`, I see only the larger rectangle. So, the larger rectangle is always there?

**Comment by asmwarrior** [comment]
> The larger rectangle (around the mpWindow) and the margin rectangle are drawn if the checkbox "Draw Box" is checked. 

Thanks for the information. I see that option now.


> In fact, only margin rectangle is always drawn but if the checkbox "Draw Box" is unchecked, the pen colour is transparent, so we don't see it, only his inside with the brush colour.

Sorry, I don't understand this sentence.  Can you explain it more?

In my test, if "Draw Box" is unchecked, both the inner rectangle(margin rectangle), and the outer rectangle(around the mpWindow) get disappeared.

**Comment by GitHubLionel** [comment]
Yes, the inner rectangle is always drawn because it use the brush to have colour in plot area. But the pen (the border of the rectangle) is transparent if "Draw box" is unchecked so it visually disappear.
